conversion/conversion.go

```diff
@@ -292,10 +296,32 @@ func processDataWithDataproc(sourceProfile profiles.SourceProfile, targetProfile
 			return err
 		}
 
-		id, err := dproc.TriggerDataprocTemplate(batchClient, srcTable, srcSchema.Schema, strings.Join(primaryKeys, ","), dataprocRequestParams)
+		op, err := dproc.TriggerDataprocTemplate(ctx, batchClient, srcTable, srcSchema.Schema, strings.Join(primaryKeys, ","), dataprocRequestParams)
+		if err != nil {
+			logger.Log.Error(fmt.Sprintf("Failing to trigger Dataproc template for %s.%s", srcSchema.Schema, srcTable))
+			return err
+		}
+		meta, _ := op.Metadata()
+		batchName := meta.Batch
```

Comment:
nit: if the variables are not used anywhere else, merge these into 1 line

Reply:
Since `op.Metadata()` returns two values, had to assign it to a variable first.